server/game/drawcard.js

```diff
@@ -93,6 +96,26 @@ class DrawCard extends BaseCard {
         });
     }
 
+    applyAttachmentBonus() {
+        let militaryBonus = parseInt(this.cardData.military_bonus);
+        if(militaryBonus) {
+            this.persistentEffect({
+                condition: () => true,
+                match: (card) => card === this.parent,
+                targetController: Players.Any,
+                effect: AbilityDsl.effects.modifyMilitarySkill(militaryBonus)
+            });
+        }
+        let politicalBonus = parseInt(this.cardData.political_bonus);
+        if(politicalBonus) {
+            this.persistentEffect({
+                condition: () => true,
+                match: (card) => card === this.parent,
+                targetController: Players.Any,
+                effect: AbilityDsl.effects.modifyPoliticalSkill(politicalBonus)
+            });
+        }
+    }
```

Comment:
@jeremylarner Did you see this part, does this look reasonable for creating attachment bonuses as effects?

Reply:
Yes, this looks fine